prac_09/cleanup_files.py

```diff
@@ -50,10 +50,24 @@ def main():
 def get_fixed_filename(filename):
     """Return a 'fixed' version of filename."""
     # First, replace the spaces and .TXT (the easy part)
-    filename = filename.replace(" ", "_").replace(".TXT", ".txt")
-
-    new_name = ""
-    # TODO: step-by-step, consider the problem cases and solve them
+    name = ""
+    for i, letter in enumerate(filename):
+        letter_check_lower = letter.islower()
```

Comment:
These are not good variables. You don't need them - just use the function - and they're Booleans, so name them like so (e.g. "is_lower").